color/api/color.api

```diff
@@ -5,8 +5,8 @@ public final class com/juul/krayon/color/Color {
 	public static fun constructor-impl (I)I
 	public static fun constructor-impl (III)I
 	public static fun constructor-impl (IIII)I
-	public static final fun copy-b4wfQcs (IIIII)I
-	public static synthetic fun copy-b4wfQcs$default (IIIIIILjava/lang/Object;)I
+	public static final fun copy-VL1fjKE (IIIII)I
+	public static synthetic fun copy-VL1fjKE$default (IIIIIILjava/lang/Object;)I
```

Comment:
@cedrickcooke do you know if this is due to the Kotlin version bump?
If so, does this mean that every time we bump the Kotlin compiler version that we're binary incompatible with previous versions?

Reply:
I believe this is a rare occurence that happened because `Color` is an inline class (which is a pre-release feature). There was a change in the name mangling scheme, I don't think there is any plan for this to happen again.

Source: https://kotlinlang.org/docs/inline-classes.html#mangling

Reply:
Oh, sweet. Sounds good. Thanks for explanation and link. 👍